src/enums/artfiact.ts

```diff
@@ -0,0 +1,25 @@
+export enum ArtifactDEMType {
+  MODEL = 'MODEL',
+  METADATA = 'METADATA',
+  ROI = 'ROI',
+  LEGEND = 'LEGEND',
+}
+
+export enum ArtifactRasterType {
```

Comment:
there is a correlation with **link type**.
Should think where to store such types